conf/msg_conf/map_msg.conf

```diff
@@ -720,6 +720,15 @@
 695: Rebellion
 696: Oktoberfest
 697: Summoner
+698: Baby Summoner
+699: Baby Ninja
+700: Baby Kagerou
+701: Baby Oboro
+702: Baby Taekwon
+703: Baby Star Gladiator
+704: Baby Soul Linker
+705: Baby Gunslinger
+706: Baby Rebellion
```

Comment:
i guess i miss look the existance of VIP atcommand messages.. perhaps maybe not all language have those id 700+ contribute to my miss-look.. i will commit another changes..